packages/create-sitecore-jss/src/templates/nextjs-personalize/src/lib/middleware/plugins/personalize.ts

```diff
@@ -10,15 +10,20 @@ class PersonalizePlugin implements MiddlewarePlugin {
   order = 1;
 
   constructor() {
+    const timeout = process.env.NEXT_PUBLIC_PERSONALIZE_TIMEOUT
+      ? ((process.env.NEXT_PUBLIC_PERSONALIZE_TIMEOUT as unknown) as number)
+      : 170;
```

Comment:
I think it's better to keep the default value in the `personalize-middleware` class, and here we just need to provide an environment variable